src/swapnaWJun20/MobileNoValidity.java

```diff
@@ -0,0 +1,64 @@
+package swapnaWJun20;
+
+public class MobileNoValidity {
+
+	boolean flag;
+
+	boolean checkAlphabet(String mobNo) {
+		flag = true;
+		for (int i = 0; i < mobNo.length(); i++) {
+			if (Character.isAlphabetic(mobNo.charAt(i)))
+				flag = false;
+		}
+		return flag;
+	}
+
+	void checkMobileNo(String mobNo) {
+		if (mobNo.length() > 14 || mobNo.length() < 10) {
+			System.out.println("\n" + mobNo + " : Mobile no is invalid.");
+		} else {
+			if (mobNo.length() == 10 && mobNo.startsWith("9") || mobNo.startsWith("8") || mobNo.startsWith("7")) {
+				checkAlphabet(mobNo);
```

Comment:
Here, in the If condition, we can use directly the method calling as you are returning the value and using that. One checkAlphabet(mobNo) calling and if(flag == true) not required together.

Reply:
Updated the code as per the comments.